pylutron/__init__.py

```diff
@@ -162,10 +163,13 @@ def _main_loop(self):
           line = t.read_until(b"\n", timeout=3)
         else:
           raise EOFError('Telnet object already torn down')
-      except (EOFError, TimeoutError, socket.timeout, AttributeError):
```

Comment:
given this seems to throw infinite random errors, I am a bit worried about removing the catch for AttributeError

could just change it to still catch and log these? i only removed it because it was suppressing a real code error

Reply:
Yeah I'd just say catch but  log them. Same applies for like 83?

Reply:
Do you think we still need AttributeError? If so I'll add it just to this clause. I added the logging though.